microsetta_interface/implementation.py

```diff
@@ -1155,6 +1163,21 @@ def post_remove_source(*,
     return _refresh_state_and_route_to_sink(account_id)
 
 
+# same as above.
+def post_request_account_removal(*, account_id):
+    # PUT is used to add the account_id to the queue
```

Comment:
confirm

Reply:
confirmed this is the endpoint called by the 'Delete Account' button in Accounts Details page.